owncloudData/src/main/java/com/owncloud/android/data/authentication/datasources/implementation/OCRemoteAuthenticationDataSource.kt

```diff
@@ -60,7 +59,7 @@ class OCRemoteAuthenticationDataSource(
 
         executeRemoteOperation {
             GetRemoteUserInfoOperation().execute(client)
-        }.let { userInfo = remoteUserInfoMapper.toModel(it)!! }
+        }.let { userInfo = it.toModel() }
```

Comment:
I'd use `toDomain` instead since `toModel` is ambiguous , we can have data model, domain model, UI model...